Microsoft.ML.FairLearn/api-example.cs

```diff
@@ -0,0 +1,52 @@
+// See https://aka.ms/new-console-template for more information
+using Microsoft.ML;
+using Microsoft.ML.AutoML;
+using Microsoft.ML.FairLearn;
+
+Console.WriteLine("Hello, World!");
+
+IDataView testData = null;
+var context = new MLContext();
+
+// train a model using fair learn
+// create a fairlearn experiment using AutoML
+var experiment = context.Auto().CreateExperiment();
+
+// reducing demographic parity using "Zone" as sensitive feature.
+// 0.9: differenceBound
+// 0.5: constraint weight
+// 10: grid limit
+// 100: grid size
+// "Zone": Sensitive feature
+experiment.ReduceDemographicParityUsingGridSearch(0.9, 0.5, 10, 100, "Zone");
```

Comment:
The version below is preferable IMO since it makes the choices explicit. This one hides the choices a bit. I'm not sure if you meant to provide both, of course, but in the fairness domain I've found it to be best to avoid providing shortcuts and always make people state choices explicitly.

Reply:
Sounds good, we will go second route then